bl/evaluation/printer.go

```diff
@@ -209,6 +209,11 @@ func GetWarningExtraMessages(invalidFile *extractor.InvalidFile) []printer.Extra
 			Text:  "Are you trying to test a raw helm file? To run Datree with Helm - check out the helm plugin README:\nhttps://github.com/datreeio/helm-datree",
 			Color: "cyan",
 		})
+	} else if IsKustomizationFile(invalidFile.Path) {
+		extraMessages = append(extraMessages, printer.ExtraMessage{
+			Text:  "it seems your trying to test a Kustomize file, Please try using `datree kustomize test`",
```

Comment:
When documentation will be ready - let's add the docs url here.

@hadar-co FYI